packaging/osx/sharedframework/shared-framework-distribution-template.xml

```diff
@@ -26,4 +26,19 @@
     <pkg-ref id="{SharedFxComponentId}.pkg">{SharedFxComponentId}.pkg</pkg-ref>
     <pkg-ref id="{HostFxrComponentId}.pkg">{HostFxrComponentId}.pkg</pkg-ref>
     <pkg-ref id="{SharedHostComponentId}.pkg">{SharedHostComponentId}.pkg</pkg-ref>
+    <installation-check script="openSSLCheck()" />
+    <script><![CDATA[
+        function openSSLCheck()
+        {
+            var openSSLRoot = "/usr/local/opt/openssl/lib/";
```

Comment:
Yes. In general, there are two ways to do this:

1. Have a `preinstall` script. This would allow, I believe, a workaround by setting an env variable. However, there is no way to define the error message, so the user would see the generic "installation failed" and would be lost *what* failed. There is a way to see that a preinstall script failed by pressing `Cmd-L` while the installer window is open...but even that doesn't give you a decent message.
2. Do what I did here, which will give you a decent message. 

So the solution is:

1. We remove the env var from postinstall, so we are unforgiving there as well.
2. We remove the check completely; this will get us back to our current state of allowing the install but failing if people try to use .NET Core. 

Thoughts?